### PR TITLE
feat: add info for all centres 

### LINKED ISSUE #163: Issues in Contact Us
**Describe the bug**
Contact US On Navigation Drawer has following error
1. Toolbar Back press not working
2. Toolbar name is wrong
3. Toll free number is not cliclacble
4. Address has for some text issues, where characters cant be displayed
5. I think it needs update for all the centres of CB

**To Reproduce**
1. Go to Contact us from nav drawer

**Expected behavior**
The above should not be their

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by mannprerak2**
Fixes #163 

Fixed all errors in this issue.

The address Text issue was due to the presence of some different encoded characters
possibly due to copy-pasting issues.

### CODE REVIEW

File: app/src/main/java/com/codingblocks/cbonlineapp/activities/AboutActivity.kt [hunk 1]
```diff
@@ -18,10 +20,35 @@ class AboutActivity : AppCompatActivity() {
         supportActionBar?.setDisplayHomeAsUpEnabled(true)
 
         callTv.setOnClickListener {
-            makeCall("18002744504")
+            val intent = Intent(Intent.ACTION_DIAL, Uri.parse("tel:18002744504"))
```
Comment (aggarwalpulkit596) [inline app/src/main/java/com/codingblocks/cbonlineapp/activities/AboutActivity.kt]: Use anko intents

File: app/src/main/res/layout/activity_about.xml [hunk 2]
```diff
@@ -115,7 +115,7 @@
...
                     android:lineSpacingExtra="3dp"
-                    android:text="1st Floor,
47, Nishant Kunj,
Main Pitampura Road,
New Delhi

Near Kohat Enclave Metro Station
(Exit from Gate # 2)
(Opposite Pillar 337)"
+                    android:text="1st Floor, 47, Nishant Kunj, Main Pitampura Road, New Delhi Near Kohat Enclave Metro Station(Exit from Gate # 2)(Opposite Pillar 337)"
```
Comment (aggarwalpulkit596) [inline app/src/main/res/layout/activity_about.xml]: Move this to strings.xml

**Comment by aggarwalpulkit596** [comment]
Add a screenshot as well

File: app/src/main/java/com/codingblocks/cbonlineapp/activities/AboutActivity.kt [hunk 3]
```diff
@@ -20,28 +20,31 @@ class AboutActivity : AppCompatActivity() {
...
 
-        email1.setOnClickListener {
+        email1_1.setOnClickListener {
             email(mailTv.text.toString())
+
         }
     }
 
+    fun dialPhone(s: String) {
+        val intent = Intent(Intent.ACTION_DIAL, Uri.parse("tel:${s}"))
```
Comment (aggarwalpulkit596) [inline app/src/main/java/com/codingblocks/cbonlineapp/activities/AboutActivity.kt]: I told you to use Anko intent use that

File: app/src/main/res/layout/activity_about.xml [hunk 4]
```diff
@@ -191,10 +191,337 @@
...
+
+                <TextView
+                    android:id="@+id/textView2_21"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="32dp"
+                    android:layout_marginTop="16dp"
+                    android:layout_marginEnd="32dp"
+                    android:lineSpacingExtra="3dp"
+                    android:text="Coding Blocks Noida, First Floor, A-73, Sector-2 (50m from Sector-15 Metro Station), Noida - 201301"
```
Comment (aggarwalpulkit596) [inline app/src/main/res/layout/activity_about.xml]: move these to strings.xml and margin to dimens.xml

**Comment by mannprerak2** [comment]
1. There is no anko intent wrapper for dialing a number(ACTION_DIAL)
(the makeCall method isn't working as it requires call permissions (uses ACTION_CALL))

2. Should I move all addresses to strings.xml?

File: app/src/main/res/layout/activity_about.xml (hunk 4 above)
Comment (mannprerak2) [reply app/src/main/res/layout/activity_about.xml]: okay

File: app/src/main/java/com/codingblocks/cbonlineapp/activities/AboutActivity.kt (hunk 3 above)
Comment (mannprerak2) [reply app/src/main/java/com/codingblocks/cbonlineapp/activities/AboutActivity.kt]: No Anko intent for dialing a number
makeCall isn't working due to call permission
https://github.com/Kotlin/anko/wiki/Anko-Commons-%E2%80%93-Intents

**Comment by mannprerak2** [comment]
[
Screenshot_20190515-232315
](url)

**Comment by aggarwalpulkit596** [comment]
> 1. There is no anko intent wrapper for dialing a number(ACTION_DIAL)
>    (the makeCall method isn't working as it requires call permissions (uses ACTION_CALL))
> 2. Should I move all addresses to strings.xml?

yes move all strings and dimens to strings.xml and dimen.xml

**Comment by mannprerak2** [comment]
Cool :smile: ?

**Pull request merged**